src/main/java/com/zendesk/maxwell/MaxwellReplicator.java

```diff
@@ -202,6 +206,8 @@ private RowMapBuffer getTransactionRows() throws Exception {
 				continue;
 			}
 
+			setReplicatorPosition((AbstractRowEvent) v4Event);
```

Comment:
is it ok to be before the switch block? (noticed there are some return statements within it)